src/referral/serializers/referral_invite_serializer.py

```diff
@@ -0,0 +1,34 @@
+from rest_framework.serializers import (
+    ModelSerializer,
+    SerializerMethodField,
+    ValidationError,
+)
+
+from referral.models import ReferralInvite
+from researchhub.settings import TESTING
+
+
+class ReferralInviteSerializer(ModelSerializer):
+    class Meta:
+        model = ReferralInvite
+        fields = [
+            "id",
+            "inviter",
+            "recipient",
+            "recipient_email",
+            "invite_type",
+            "created_date",
+            "recipient_email",
+            "referral_first_name",
+            "referral_last_name",
+            "unified_document",
+        ]
+
+    def create(self, validated_data):
+        data = validated_data
+        instance = ReferralInvite.create(**data)
+
+        if not TESTING:
+            instance.send_invitation()
+
+        return instance
```

Comment:
Overriding the default create isn't necessary here. You can call `send_invitation` once the serializer has created the object. This can be done with `serializer.instance.send_invitation` in the viewset.

Reply:
Are you referring to the create method of the view?
Where should I call `ReferralInvite.create(**data)` If I don't have view method to call it from?